maro/cli/grass/node.py

```diff
@@ -50,8 +53,33 @@ def stop_node(cluster_name: str, replicas: int, node_size: str, **kwargs):
 def list_node(cluster_name: str, **kwargs):
     cluster_details = load_cluster_details(cluster_name=cluster_name)
 
-    if cluster_details["mode"] == "grass/azure":
+    if cluster_details["mode"] == "grass/azure" or "grass/on-premises":
         executor = GrassAzureExecutor(cluster_name=cluster_name)
         executor.list_node()
-    else:
-        raise BadRequestError(f"Unsupported command in mode '{cluster_details['mode']}'.")
+
+
+def node_join(node_join_path: str, **kwargs):
```

Comment:
Because node join action doesn't need cluster_name as parameter, cluster name already included in YAML file. 
It only needs pass a YAML file as parameter.